mapper/sequence_mapper.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+class SequenceMapper(object):
+    """
+    Provides the logic for mapping the sequence number from one message set to
+    another.
+    """
+    def map(self, from_messageset, to_messageset, sequence):
```

Comment:
This doesn't do much, is it a placeholder for logic that will be added later?

Reply:
Yup. When the gates message sets have been finalised, then we can add the mapping logic here on how to go back and forth between the two sets.

Reply:
Cool, then I'm +1